src/core_plugins/kibana/public/visualize/listing/visualize_listing.html

```diff
@@ -0,0 +1,172 @@
+<!-- Local nav. -->
+<kbn-top-nav name="visualize">
+  <!-- Transcluded elements. -->
+  <div data-transclude-slots>
+    <!-- Title. -->
+    <div
+      data-transclude-slot="topLeftCorner"
+      class="kuiLocalTitle"
+    >
+      Visualize
+    </div>
+  </div>
+</kbn-top-nav>
+
+<div class="kuiViewContent kuiViewContent--constrainedWidth">
+  <!-- ControlledTable -->
+  <div class="kuiViewContentItem kuiControlledTable kuiVerticalRhythm">
+    <!-- ToolBar -->
+    <div class="kuiToolBar">
+      <div class="kuiToolBarSearch">
+        <div class="kuiToolBarSearchBox">
+          <div class="kuiToolBarSearchBox__icon kuiIcon fa-search"></div>
+          <input
+            class="kuiToolBarSearchBox__input"
+            type="text"
+            placeholder="Search..."
+            aria-label="Filter"
+            ng-model="listingController.filter"
+          >
+        </div>
+      </div>
+
+      <div class="kuiToolBarSection">
+        <!-- Bulk delete button -->
+        <button
+          class="kuiButton kuiButton--danger"
+          aria-label="Delete selected objects"
+          ng-if="listingController.getSelectedItemsCount() > 0"
+          ng-click="listingController.deleteSelectedItems()"
+          tooltip="Delete selected visualizations"
+        >
+          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-trash"></span>
+        </button>
+
+        <!-- Create visualization button -->
+        <a
+          class="kuiButton kuiButton--primary"
+          href="#/visualize/new"
+          aria-label="Create new visualization"
+          ng-if="listingController.getSelectedItemsCount() === 0"
+          tooltip="Create new visualization"
+        >
+          <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-plus"></span>
+        </a>
+      </div>
+
+      <div class="kuiToolBarSection">
+        <!-- Pagination -->
+        <tool-bar-pager-text
+          start-item="listingController.pager.startItem"
+          end-item="listingController.pager.endItem"
+          total-items="listingController.pager.totalItems"
+        ></tool-bar-pager-text>
+        <tool-bar-pager-buttons
+          has-previous-page="listingController.pager.hasPreviousPage"
+          has-next-page="listingController.pager.hasNextPage"
+          on-page-next="listingController.onPageNext"
+          on-page-previous="listingController.onPagePrevious"
+        ></tool-bar-pager-buttons>
+      </div>
+    </div>
+
+    <!-- NoResults -->
+    <div class="kuiPanel kuiPanel--centered" ng-if="!listingController.items.length">
```

Comment:
Use the new `Prompt For Items` section?  Also for some reason this isn't coming out as styled like the dashboard one, I think kuiNoResults should be kuiNoItems.

<img width="1126" alt="screen shot 2017-02-06 at 10 00 38 am" src="https://cloud.githubusercontent.com/assets/16563603/22652108/21a759a6-ec53-11e6-84c0-391a8d0943a1.png">